OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -434,6 +434,9 @@ void MetaModelEditor::addInterfacesData(QDomElement interfaces)
   QDomNodeList subModelList = mXmlDocument.elementsByTagName("SubModel");
   for (int i = 0 ; i < subModelList.size() ; i++) {
     QDomElement subModel = subModelList.at(i).toElement();
+    while(!subModel.firstChildElement("InterfacePoint").isNull()) { //First remove all existing interfaces!
```

Comment:
You don't need to do that. That's a very bad idea.
I already updated the code yesterday to handle both cases i.e., update interface point if its changed OR insert interface point if we don't have any.